### ISSUE TITLE
WARNING (in the IsOption instance for AcceptTests)

### ISSUE THREAD

**Issue by sjakobi** [phile314/tasty-silver#25]
I'm working on using `tasty-silver` in a test suite. I'm currently using `Test.Tasty.Silver.goldenVsAction`. When I run the testsuite, the following warning is printed:

```
WARNING (in the IsOption instance for AcceptTests):
Using default values (e.g., with Options.Applicative.value) in
optionCLParser is prohibited, as it interferes with tasty's ability
to read environment variable options properly. Moreover, assigning
default values is unnecessary, as their functionality is subsumed
by the defaultValue method of IsOption.
```

**Comment by phile314** [comment]
Released as v3.2.1.

**Comment by sjakobi** [comment]
Thanks a lot, @phile314! :)
